common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -92,8 +92,7 @@ private static void handleExceptionInEventListener(Throwable t) {
                         + "' has crashed and will be disabled")
                 .withStyle(ChatFormatting.RED));
 
-        feature.setUserEnabled(false);
-        feature.tryUserToggle();
+        feature.disable();
```

Comment:
I don't what is wrong here as of now, but @magicus likely has a deeper, correct vie

Reply:
What do you mean is wrong? As in it does not work as expected, or as in the code does not look like you think?

Reply:
It has been reverted